src/components/App.js

```diff
@@ -4,10 +4,10 @@ import ButtonPanel from './ButtonPanel';
 
 function App() {
   return (
-    <div className="App">
-      <Display />
+    <>
+      <Display previousVal="" currentVal="0" />
```

Comment:
- [ ] <b>OPTIONAL:</b> You have already set default values in `Display` component in `Display.defaultProps`, that means, you don't have to do it here. since this wasn't mentioned in the previous review, you can consider it as optional, but I strongly recommend you to fix it.